third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-cleric-prophecy.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+  <info>
+		<author url="https://www.arcanumworlds.com/">Arcanum Worlds</author>
+		<update version="0.0.1">
+			<file name="subclass-cleric-prophecy.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/arcanum-worlds/odyssey-of-the-dragonlords/subclass-cleric-prophecy.xml" />
+		</update>
+	</info>
+  <element name="Prophecy Domain" type="Archetype" source="Odyssey of the Dragonlords" id="ID_AW_OOTD_ARCHETYPE_CLERIC_PROPHECY_DOMAIN">
+    <supports>Divine Domain</supports>
+    <description>
+      <p>The Prophecy domain focuses on foretelling danger and preventing harmful outcomes. Clerics of this domain are also known as “oracles.” Oracles pay special attention to dreams, intuitive thoughts, and hallucinatory visions. They are widely believed to be capable of predicting the future, though their visions are often misinterpreted. They live by the phrase, “know thyself.”</p>
+      <p>Oracles often carry special varieties of incense and holy water with them. The holy water may be boiled to produce vapors, which the oracle inhales in order to enter a trance-like state. During such a trance, the oracle thrashes and wails as they experience flashes of divine insight. They often employ scribes to write down their words, so that they may be recalled later.</p>
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_DOMAIN_SPELLS" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_MINDFUL_SENSES" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_BLESSING_OF_FORESIGHT" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_CD_PROPHETIC_TRANCE" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_VISIONS_OF_DANGER" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_HEALING_VAPORS" />
+      <div element="ID_AW_OOTD_ARCHETYPE_FEATURE_PROPHECY_DOMAIN_PERFECT_FORESIGHT" />
+    </description>
+    <sheet>
+      <description>The gods of the forge are patrons of artisans who work with metal, from a humble blacksmith who keeps a village in horseshoes and plow blades to the mighty elf artisan whose diamond-tipped arrows of mithral have felled demon lords.</description>
+    </sheet>
```

Comment:
some copy/paste left over from another archetype

all archetype sheet descriptions can be set to `display="false"` as they will no longer be shown in the features section to save space